decimal64const.go

```diff
@@ -53,3 +53,6 @@ var Min64 = newFromParts(0, -398, 1)
 
 var zeroes = []Decimal64{Zero64, NegZero64}
 var infinities = []Decimal64{Infinity64, NegInfinity64}
+
+// defaultContext is the context that Arithmetic functions will use in order to do calculations
+var defaultContext = Context64{roundHalfUp}
```

Comment:
Is it worth exporting this? I'm torn, since global state is usually an anti-pattern, but if a particular program needs consistent non-standard rounding everywhere, the ability to change the default might be quite valuable.

Reply:
I think it'd be better to export it, then stick a massive "warning" in the documentation about changing it